.agents/skills/add-locale-batch/scripts/locale_batch_status.py

```
@@ -0,0 +1,121 @@
+from typing import Any
+
+
+def run_json(args: list[str]) -> Any:
+    completed = subprocess.run(args, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    if completed.returncode != 0:
+        raise RuntimeError(f"command failed: {' '.join(args)}\n{completed.stderr.strip()}")
+    return json.loads(completed.stdout or "null")
+
+
+def run_text(args: list[str]) -> str:
+    completed = subprocess.run(args, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)